plugins/inputs/windows_event_log/wineventlog/utils.go

```diff
@@ -80,21 +83,11 @@ func CreateQuery(path string, levels []string) (*uint16, error) {
 }
 
 func UTF16ToUTF8Bytes(in []byte, length uint32) ([]byte, error) {
```

Comment:
Does this function get called externally anymore after your changes? Should this be made private scoped?